bff-service/scripts/create.sh

```diff
@@ -0,0 +1 @@
+eb create devel --single --cname constantineisaev-bff-serivce-dev 
```

Comment:
`npm run ...` would be better since it could be used across different OS w/o extra efforts.